bindings/buffer.hh

```diff
@@ -0,0 +1,70 @@
+#pragma once
```

Comment:
minor: what should headers look like ?

Reply:
Don't know, should I use the same one as ddprof ?